utils/wasm-gen/src/config/module.rs

```diff
@@ -307,34 +326,45 @@ pub struct ConstantParams {
     tail_call_enabled: bool,
     relaxed_simd_enabled: bool,
     saturating_float_to_int_enabled: bool,
-    sign_extension_enabled: bool,
+    sign_extension_ops_enabled: bool,
     simd_enabled: bool,
-    float_enabled: bool,
-    memory_grow_enabled: bool,
+    allow_floats: bool,
     min_types: usize,
-    memory_offset_choices: (u32, u32, u32),
+    memory_offset_choices: MemoryOffsetChoices,
+    gc_enabled: bool,
+    custom_page_sizes_enabled: bool,
+    generate_custom_sections: bool,
+    exports: Option<Vec<u8>>,
+    shared_everything_threads_enabled: bool,
+    allow_invalid_funcs: bool,
+    wide_arithmetic_enabled: bool,
+    extended_const_enabled: bool,
+    // our patches
     reserved_memory_size: Option<u64>,
+    memory_grow_enabled: bool,
 }
 
 impl Default for ConstantParams {
     fn default() -> Self {
         ConstantParams {
             bulk_memory_enabled: false,
-            sign_extension_enabled: false,
+            // TODO: enable
```

Comment:
reference some issue please or resolve in place: seems like not a big deal

Reply:
@techraed WDYT

Reply:
> reference some issue please or resolve in place: seems like not a big deal

#4502

Reply:
> I'd also allow some other banned stuff with low appearance rate to check that we really ban them

can be done in a separate unit test